af/fastpath/fastpath/s3feeder.py

```diff
@@ -89,8 +85,13 @@ def load_multiple(fn: str) -> Generator[MsmtTup, None, None]:
         raise RuntimeError(fn)
 
 
-def create_s3_client():
-    return boto3.Session(profile_name=AWS_PROFILE).client("s3")
+def create_s3_client(conf):
+    c = boto3.client(
+        "s3",
+        aws_access_key_id=conf.s3_access_key,
+        aws_secret_access_key=conf.s3_secret_key,
+    )
+    return c
```

Comment:
Are you accessing the s3 storage in read-only mode or do you also need to write?

If you are accessing it in read-only mode I suggest you instead use unsigned requests so that we are sure that we aren't paying for file access.

This issue provides insight on how to do that: https://github.com/ooni/backend/issues/279#issue-550694470

Reply:
👍